tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -209,3 +217,34 @@ func createDownstreamCluster(client *rancher.Client, clusterType string) (*manag
 
 	return clusterObject, steveObject, testClusterConfig, nil
 }
+
+func createGlobalRole(client *rancher.Client, inheritedClusterrole []string) (*v3.GlobalRole, error) {
+	globalRole.Name = namegen.AppendRandomString("testgr")
+	globalRole.InheritedClusterRoles = inheritedClusterrole
+	createdGlobalRole, err := rbac.CreateGlobalRole(client, &globalRole)
+	return createdGlobalRole, err
+}
+
+func createGlobalRoleAndUser(t *testing.T, client *rancher.Client, inheritedClusterrole []string) (*management.User, error) {
+	globalRole, err := createGlobalRole(client, inheritedClusterrole)
+	require.NoError(t, err)
+
+	createdUser, err := users.CreateUserWithRole(client, users.UserConfig(), standardUser, globalRole.Name)
+	require.NoError(t, err)
+
+	return createdUser, err
+
+}
+
+func getErrorMessage(err, startText, endText string) (actualError string) {
```

Comment:
Do we need this function? Can we not use `require.Contains`to check if the error message contains the text we are looking for?
For example, for the code [here](https://github.com/rancher/rancher/pull/43363/commits/8c6424c1a26e0b748c6e517251dc0cbb03a20189#diff-f6eedc49685d06b16bd2b8b68e5eb22ea009eea42d63b22cd78a873f6fa75632R172-R177), can we not implement it as: 
```
require.Contains(grw.T(), err.Error(), "admission webhook")
require.Contains(grw.T(), err.Error(), "cannot be deleted because it is inherited by globalRole(s) " + globalRole.Name)
```
**OR** if start and end of the error message matters, then, could you use regex?
```
pattern := "^admission webhook.*" + regexp.QuoteMeta("cannot be deleted because it is inherited by globalRole(s) "+globalRole.Name) + "$"
require.Regexp(grw.T(), regexp.MustCompile(pattern), err.Error())
```